bin/monit

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 
 # find out where CMSMonitoring is installed on a system
-root=`python -c "import CMSMonitoring; print '/'.join(CMSMonitoring.__file__.split('/')[:-1])"`
+root=$(python -c "import CMSMonitoring; print('/'.join(CMSMonitoring.__file__.split('/')[:-1]))")
+MONIT_HOME="$(cd "$(dirname "$0")" && pwd)"
+export MONIT_DB_DICT_FILE="$MONIT_HOME/../static/datasources.json"
```

Comment:
This is fragile, instead do something like this:
```
MONIT_DB_DICT_FILE=${MONIT_DB_DICT_FILE:-$MONIT_HOME/../static/datasources.json}
```
This will allow to specify datasources.json via environment variable and have fallback to MONIT home area (or whatever).

Reply:
Agree :+1: